manifests/r/RocketChat/RocketChat/3.5.0.0/RocketChat.RocketChat.installer.yaml

```diff
@@ -0,0 +1,40 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: RocketChat.RocketChat
+PackageVersion: 3.5.0.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/RocketChat/Rocket.Chat.Electron/releases/download/3.5.0/rocketchat-setup-3.5.0.exe
+  InstallerSha256: 0D3B5C143AC9CEE6FDF74EA2CE43623DE43E8BB7331614E4BE82F041789A9172
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://github.com/RocketChat/Rocket.Chat.Electron/releases/download/3.5.0/rocketchat-setup-3.5.0.exe
+  InstallerSha256: 0D3B5C143AC9CEE6FDF74EA2CE43623DE43E8BB7331614E4BE82F041789A9172
+  InstallerSwitches:
+    Custom: /ALLUSERS
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
```

Comment:
```suggestion
  Architecture: x64
```
This is what I see on x86:
![image](https://user-images.githubusercontent.com/11600822/132858833-71fee5a2-cd4c-4ef4-9dec-fb20bbf69c3d.png)

Reply:
Thanks for looking into it!